test/Microsoft.Dnx.Testing.CommonUtils/DnxSdk/DnxSdk.cs

```diff
@@ -0,0 +1,85 @@
+﻿using System;
+using System.Runtime.Versioning;
+
+namespace Microsoft.Dnx.Testing
+{
+    public class DnxSdk
+    {
+        public string Version { get; set; }
+
+        public string Flavor { get; set; }
+
+        public string Architecture { get; set; }
+
+        public string OperationSystem { get; set; }
+
+        public string Path { get; set; }
+
+        public string FullName { get; set; }
+
+        public FrameworkName TargetFramework { get; set; }
+
+        public Dnu Dnu => new Dnu(Path);
+
+        public Dnx Dnx => new Dnx(Path);
+
+        public static string GetRuntimeHome()
+        {
+            var homePath = Environment.ExpandEnvironmentVariables(Environment.GetEnvironmentVariable("DNX_HOME"));
```

Comment:
I think there are at least 5 places in our dnx code base where we do this. Please try to find at least one of them and reuse it here